### PR TITLE
style: Update style in configure-artifact-repository.md

### PR DESCRIPTION
**Pull request by igarciar**
change configuration of minion section

Checklist:

* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo/blob/master/USERS.md).

### CODE REVIEW

File: docs/configure-artifact-repository.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ Subsequent sections will show how to use it.
 
 ## Configuring Minio
 
-```
-$ brew install helm # mac, helm 3.x
-$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/ # official Helm stable charts
-$ helm repo update
-$ helm install argo-artifacts stable/minio --set service.type=LoadBalancer --set fullnameOverride=argo-artifacts
+```bash
+brew install helm # mac, helm 3.x
```
Comment (simster7) [inline docs/configure-artifact-repository.md:22]: Hmm, we use the `$ ...` convention across our docs (even in other places in this same file). In my opinion, what we currently have is easier to parse and understand.

Do you have a good reason to remove the `$ ...` convention?

**Review by alexec — CHANGES_REQUESTED** [review]
Thank you. We do need you to sign your commits and make the PR title semantic please.

**Comment by igarciar** [comment]
Hi,
The problem is when you copy the lines or when you used the link.

in other hand, if you need remark same code is recommended to identify the
type of code. Y mark de code like this:

----------

```*bash*

 brew install helm # mac, helm 3.x
 helm repo add stable
https://kubernetes-charts.storage.googleapis.com/ # official Helm
stable charts
 helm repo update
 helm install argo-artifacts stable/minio --set
service.type=LoadBalancer --set fullnameOverride=argo-artifacts

````

----------

Ignacio García Ruiz
ignacio.garcia.ruiz@gmail.com

**Comment by alexec** [comment]
Closing. Another PR addressed this.

**Pull request closed without merging**